uart_emu/docs/src/index.md

```diff
@@ -0,0 +1,71 @@
+# UART EMU
+
+## Allocate UART EMULATOR Object with RMT Backend
+
+## Features
+
+- **RMT-based Implementation**: Utilizes ESP32's RMT peripheral for precise timing control
+- **Configurable Baud Rate**: Supports various baud rates
+- **Asynchronous Operation**: Non-blocking transmit and receive operations
+- **Callback Support**: Event-driven programming with TX/RX completion callbacks
+- **DMA Support(need RMT Supports DMA)**: Optional DMA acceleration for improved performance
+- **Ping-Pong Buffer**: Efficient handling of continuous data streams
+
+## Current Limitations
+
+- **Data Bits**: Only 8-bit data format supported
+- **Parity**: Only no-parity mode supported
+- **Stop Bits**: 1 or 2 stop bits supported
+- **Flow Control**: No hardware flow control support
+- **Error Detection**: Limited error detection capabilities compared to hardware UART
+
+## API Documentation
+
+For detailed API documentation, please refer to the header files:
+- [uart_emu.h](./include/uart_emu.h) - Main public API
```

Comment:
After the Markdown file is compiled into html and hosted on the Github pages, user can't find this link.

Reply:
Luckily this error is also caught by the CI.